src/customers/customers.entity.ts

```diff
@@ -0,0 +1,43 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+
+@Entity('customers')
+export class Customer {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({ type: 'text' })
+  first_name: string;
+
+  @Column({ type: 'text' })
+  middle_name: string;
```

Comment:
Let's make `middle_name`, `address_1` and `address_2` optional

for `middle_name` it would look something like this:
```suggestion
  @Column({ type: 'text', nullable: true })
  middle_name: string | null;
```